nix/importTOML.nix

```diff
@@ -3,6 +3,7 @@ file:
 # Return a module that gets lib as an argument
 { lib, ... }:
 let
+  importTOML = import ./importTOML.nix;
```

Comment:
Can you bind the returned value to `importTOML` instead of re-importing the same file?

Reply:
Done in https://github.com/numtide/devshell/pull/254/commits/ef97d3f9a8932a316171bf696d4c396c713f6a40 :smile: 

I was in minimal diff mode.